tests/test_carbon_handler.cairo

```diff
@@ -109,36 +111,33 @@ fn test_set_project_value() {
 }
 
 #[test]
-fn test_get_project_value_not_set() {
-    // [Setup]
+fn test_get_project_carbon_not_set() {
     let (project_address, _) = deploy_project(c::OWNER().into());
     let project = IAbsorberDispatcher { contract_address: project_address };
-    // [Assert] default project_value is 0
-    let fetched_value = project.get_project_value();
-    assert(fetched_value == 0, 'default project_value is not 0');
+    // [Assert] default project_carbon is 0
+    let fetched_value = project.get_project_carbon();
+    assert(fetched_value == 0, 'default project_carbon is not 0');
 }
 
 #[test]
-fn test_set_project_value_twice() {
-    // [Setup]
+fn test_set_project_carbon_twice() {
```

Comment:
maybe change the name to be more behavior oriented by `test_set__and_update_project_carbon`